### ISSUE TITLE
Document Method to Uninstall Volta

### ISSUE THREAD

**Issue by chsi022** [volta-cli/volta#531]
On a Unix system, I have installed Volta via "curl https://get.volta.sh | bash" , how do I then remove Volta from the system?

**Comment by charlespierce** [comment]
Hi @chsi022, to uninstall Volta after using the default installer, you would need to do two things:

1. Remove the `~/.volta` directory
2. Edit your shell profile file (`~/.bashrc`, `~/.bash_profile`, `~/.profile` or similar) to remove the 3 lines that reference `VOLTA_HOME`

I'm going to rename this issue to note that we should document that process.

**Comment by chsi022** [comment]
Thank you! I'll keep these in mind if I need to remove Volta from the system.

**Comment by wpscholar** [comment]
The steps above didn't entirely work. Now I get `-bash: ~/.volta/bin/node: No such file or directory` when I run `node -v`.

**Comment by wpscholar** [comment]
Ideally, there should be an uninstall script.

**Comment by charlespierce** [comment]
@wpscholar Did you, by chance, open a new terminal after making the above changes? `bash` hashes and keeps the location of any command you run, which persists until you open a new terminal.

It appears you can also remove the hash for a command (like `node`), by running `hash -d node`

Agreed we should have some sort of script / command for a full uninstall, we mostly need to figure out the best way to deliver that and make it available, as well as handle as many OS / shell combinations as possible.

**Comment by wpscholar** [comment]
@charlespierce You are correct. Opening a new terminal window made that last issue go away.

**Comment by detj** [comment]
Would like to point out that I see an `uninstall` sub-command when after running `volta help`, but don't see any corresponding help in the official documentation about how this command is supposed to be used.

```sh
$ volta help
Volta 0.8.1
The JavaScript Launcher ⚡

    To install a tool in your toolchain, use `volta install`.
    To pin your project's runtime or package manager, use `volta pin`.

USAGE:
    volta [FLAGS] [SUBCOMMAND]

FLAGS:
        --verbose    Enables verbose diagnostics
        --quiet      Prevents unnecessary output
    -v, --version    Prints the current version of Volta
    -h, --help       Prints help information

SUBCOMMANDS:
    fetch          Fetches a tool to the local machine
    install        Installs a tool in your toolchain
    uninstall      Uninstalls a tool from your toolchain
    pin            Pins your project's runtime or package manager
    list           Displays the current toolchain
    completions    Generates Volta completions
    which          Locates the actual binary that will be called by Volta
    setup          Enables Volta for the current user / shell
    run            Run a command with custom Node, npm, and/or Yarn versions
    help           Prints this message or the help of the given subcommand(s)
```

https://docs.volta.sh/reference/

**Comment by charlespierce** [comment]
@detj That's a good callout, I'll update the online docs - `volta uninstall` is for uninstalling global binaries that were installed with `volta install` 😃

**Comment by charlespierce** [comment]
Fixed in volta-cli/website#82 and volta-cli/website#83
